testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -2983,6 +2983,22 @@
             "acceptation": "Introduced public ExecException",
             "changes": []
         },
+        {
+            "type": "org.gradle.testing.jacoco.tasks.JacocoReportBase",
+            "member": "Method org.gradle.testing.jacoco.tasks.JacocoReportBase.getAntBuilder()",
+            "acceptation": "Removed protected method injecting internal class",
+            "changes": [
+                "Method has been removed"
+            ]
+        },
+        {
+            "type": "org.gradle.testing.jacoco.tasks.JacocoReportBase",
+            "member": "Method org.gradle.testing.jacoco.tasks.JacocoReportBase.getCallbackActionDecorator()",
+            "acceptation": "Removed protected method injecting internal class",
+            "changes": [
+                "Method has been removed"
+            ]
+        },
```

Comment:
❓ Why is this in here?

Reply:
In addition to the other changes in this PR, we remove two unused protected methods on JacocoReportBase which return internal types.